src/assisted_test_infra/test_infra/controllers/node_controllers/terraform_controller.py

```diff
@@ -174,8 +174,11 @@ def _fill_tfvars(self, running=True):
         tfvars["bootstrap_in_place"] = self._config.bootstrap_in_place
 
         vips = self.get_ingress_and_api_vips()
-        tfvars["api_vip"] = vips["api_vip"]
-        tfvars["ingress_vip"] = vips["ingress_vip"]
+        tfvars["api_vip"] = self._config.api_vip if self._config.api_vip else vips["api_vip"]
```

Comment:
isn't it equivalent to 
```suggestion
        tfvars["api_vip"] = self._config.api_vip or vips["api_vip"]
```